src/Sulu/Component/Webspace/Tests/Unit/Url/ReplacerTest.php

```diff
@@ -17,7 +17,7 @@ class ReplacerTest extends \PHPUnit_Framework_TestCase
 {
     public function testHasLanguageReplacer()
     {
-        $replacer = new Replacer('sulu.io');
+        $replacer = new Replacer();
```

Comment:
Replacer has no constructor so it has no parameters: https://github.com/sulu/sulu/blob/1.6.7/src/Sulu/Component/Webspace/Url/Replacer.php